src/devices/shelly.ts

```diff
@@ -9,4 +9,11 @@ export const definitions: DefinitionWithExtend[] = [
         description: "Mini gen 4",
         extend: [m.onOff({powerOnBehavior: false})],
     },
+    {
+        zigbeeModel: ["1"],
```

Comment:
Could you also provide the data/database.db entry of this device? Since the zigbee model is quite short it can conflict, therefore I want to add the `manufacturerName`, e.g. 

```suggestion
        fingerprint: [{modelID: "1", "manufacturerName: "Shelly????"}],
```

Reply:
Happy to. What's the best way for me to send that @Koenkk?

Reply:
I worked it out. Here is the data you need including the headers (which I assume you know).

```

field1 field2 field3 field4 field5 field6 field7 field8 field9 field10 field11 field12 field13 field14 field15 field16 field17 field18 field19 field20 field21 field22 field23 field24 field25 field26 field27 field28 field29 field30 field31 field32 field33 field34 field35 field36 field37 field38 field39 field40 field41 field42 field43 field44 field45 field46 field47 field48 field49 field50 field51 field52 field53 field54 field55 field56 field57 field58 field59 field60 field61 field62 field63 field64 field65 field66 field67 field68 field69 field70 field71 field72 field73 field74 field75 field76 field77 field78 field79 field80 field81 field82 field83 field84 field85 field86 field87 field88 field89 field90 field91 field92 field93 field94 field95 field96 field97 field98 field99 field100 field101 field102 field103 field104 field105 field106 field107 field108 field109 field110 field111 field112 field113 field114 field115 field116 field117 field118 field119 field120 field121 field122 field123 field124 field125 field126 field127 field128 field129 field130 field131 field132 field133 field134 field135 field136 field137 field138 field139 field140 field141 field142 field143 field144 field145 field146 field147 field148 field149 field150
{id:24 type:Router ieeeAddr:0xccba97fffec87d4c nwkAddr:61538 manufId:5264 manufName:Shelly powerSource:Mains (single phase) modelId:1 epList:[1 239 242] endpoints:{1:{profId:260 epId:1 devId:2 inClusterList:[0 3 4 5 6] outClusterList:[] clusters:{genBasic:{attributes:{}} genOnOff:{attributes:{onOff:0}}} binds:[{cluster:6 type:endpoint deviceIeeeAddress:0x00124b0025e1f180 endpointID:1}] configuredReportings:[{cluster:6 attrId:0 minRepIntval:0 maxRepIntval:65000 repChange:1}] meta:{}} 239:{profId:49153 epId:239 devId:8193 inClusterList:[64513 64514] outClusterList:[] clusters:{} binds:[] configuredReportings:[] meta:{}} 242:{profId:41440 epId:242 devId:97 inClusterList:[] outClusterList:[33] clusters:{} binds:[] configuredReportings:[] meta:{}}} zclVersion:8 interviewCompleted:true meta:{configured:332242049} lastSeen:1743406694644}

```